app/ipc-types.ts

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable quotes */
+
+export enum WorkerStatus {
+  runningInBackground = `RUNNING_IN_BACKGROUND`,
+  stopped = `STOPPED`,
```

Comment:
I'm curious why this enum is snake cased? all other are camel cased and that is more common. I'm assuming it has to do with the type union on `GlobalStatus`, is that a thing from electron?

Reply:
Initially, because `@typescript-eslint/naming-convention` complained! But when I moved it into this file I had to disable that anyway because of the other enums,  so you're right it should match the others.